packages/eslint-config-node/readme.md

```diff
@@ -28,6 +28,46 @@ Use this ruleset in conjunction with any of the above version-specific rulesets.
 
 This ruleset includes rules which deal with how the code looks like and not how it works. It helps keeping the code clean and consistent. 🎨
 
+## STRV Config
+
+package.json
+
+```json
+{
+  "eslintConfig": {
+    "extends": [
+      "@strv/eslint-config-node/<v10 or v12>",
+      "@strv/eslint-config-node/optional",
+      "@strv/eslint-config-node/style"
+    ]
+  }
+}
+```
+
+.eslintrc or .eslintrc.json
```

Comment:
Let's switch to _.*rc.js_ examples wherever supported. 🙏 You can have comments in JS files and you can even add some tiny amount of scripting/dynamic behaviour if you really really need to.